build-support/bin/cherry_pick.sh

```diff
@@ -60,7 +61,8 @@ if [[ -z $CATEGORY_LABEL ]]; then
   echo "Couldn't detect category label on PR. What's the label? (E.g., category:bugfix)"
   read -r CATEGORY_LABEL
 fi
-REVIEWERS=$(gh pr view "$PR_NUM" --json reviews --jq '.reviews.[].author.login' | sort | uniq)
+REVIEWERS=$(gh pr view "$PR_NUM" --json author --jq '.author.login')
```

Comment:
I just tried, adding a PR author as a reviewer of their own PR is a no-op, so this is safe to do unconditionally.